src/Build/BackEnd/Components/Scheduler/Scheduler.cs

```diff
@@ -1976,14 +1977,14 @@ private bool CheckIfCacheMissOnReferencedProjectIsAllowedAndErrorIfNot(int nodeF
                             NewBuildEventContext(),
                             MessageImportance.Normal,
                             "SkippedConstraintsOnRequest",
-                            configs.parentConfig.ProjectFullPath,
-                            configs.requestConfig.ProjectFullPath);
+                            configs.ParentConfig.ProjectFullPath,
+                            configs.RequestConfig.ProjectFullPath);
                 }
 
                 return true;
             }
 
-            var (requestConfig, parentConfig) = GetConfigurations();
+            (BuildRequestConfiguration requestConfig, BuildRequestConfiguration parentConfig) = GetConfigurations();
```

Comment:
I am 100% sold on automatic `var` removal. Lets discuss it.

Reply:
I removed it to abide by the style guidelines in `.editorconfig`: https://github.com/dotnet/msbuild/blob/main/.editorconfig#L46
Since I'm the last person to touch this code, I thought it would be a good idea to fix style issues as well.

Reply:
OK. No big deal.

Reply:
I prefer no var, personally, and I think we talked about it a while ago and collectively decided on removing var.